admin-dev/themes/new-theme/js/components/pagination/dynamic-paginator.ts

```diff
@@ -290,15 +291,14 @@ export default class DynamicPaginator {
   }
 
   /**
-   * @param {Object} selectorsMap
+   * @private
    */
-  setSelectorsMap(selectorsMap: Record<string, string>): void {
-    if (selectorsMap) {
-      this.selectorsMap = selectorsMap;
-
+  private setSelectorsMap(): void {
+    if (Object.keys(this.selectorsMap).length !== 0) {
```

Comment:
Although one thing that is bugging me, with this way of checking it means as long as you have one selector overridden it won't use the default ones at all, but it doesn't mean you will have all of them and it could result in bugs on some of the inputs

Or said differently, if you want to change ONE selector without breaking the component you need to make sure to override them ALL

Reply:
I know it was not in the initial scope but maybe this would be the chance to improve this? By checking the selectors one by one?

Reply:
Suggestion to handle this:
```javascript
private setSelectorsMap(Record<string, string> inputSelectors): void {
   this.selectorsMap = {
      jumpToPageInput: 'input[name="paginator-jump-page"]',
      firstPageBtn: 'button.page-link.first',
      firstPageItem: 'li.page-item.first',
      nextPageBtn: 'button.page-link.next',
      nextPageItem: 'li.page-item.next',
      previousPageBtn: 'button.page-link.previous',
      previousPageItem: 'li.page-item.previous',
      lastPageItem: 'li.page-item.last',
      lastPageBtn: 'button.page-link.last',
      pageLink: 'button.page-link',
      limitSelect: '#paginator-limit',
      paginationInfoLabel: '#pagination-info',
      ...inputSelectors
    };
}
```